objects/official/footpath_surface/openrct2.footpath_surface.invisible.json

```
@@ -8,7 +8,7 @@
     "noSlopeRailings": true
   },
   "images": [
-    "", "", "", "", "", "", "", "", "", "", "", "", "", "", "", "", "", "", "", "", "", "", "", "", "", "", "", "", "", "", "", "", "", "", "", "", "", "", "", "", "", "", "", "", "", "", "", "", "", "", ""
+    "", "", "", "", "", "", "", "", "", "", "", "", "", "", "", "", "", "", "", "", "", "", "", "", "", "", "", "", "", "", "", "", "", "", "", "", "", "", "", "", "", "", "", "", "", "", "", "", "", "", "", ""
```

The images array basically holds two parts, the preview and the actual sprites. There are 51 sprites for normal paths, this change makes it look like there are 52. No big deal, but I think it would be cleaner to explicitly state that the preview is blank.

```suggestion
    "",
    "", "", "", "", "", "", "", "", "", "", "", "", "", "", "", "", "", "", "", "", "", "", "", "", "", "", "", "", "", "", "", "", "", "", "", "", "", "", "", "", "", "", "", "", "", "", "", "", "", "", ""
```